components/error/errorCard.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div
+    class="text-center border-1px pa-10"
+    style="

```

Comment:
Inline styles like this are fine for only adding one or two attributes. But for adding a lot of styles like this you should make a class and place the styles in the `<style scoped></style>` tag.